Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -832,7 +826,7 @@
 #define ENABLE_ARKIT_INLINE_PREVIEW 1
 #endif
 
-#if (ENABLE(ARKIT_INLINE_PREVIEW_MAC) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 120300) || (ENABLE(ARKIT_INLINE_PREVIEW_IOS) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 150400)
+#if (ENABLE(ARKIT_INLINE_PREVIEW_MAC) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 120300) || ENABLE(ARKIT_INLINE_PREVIEW_IOS)
```

Comment:
Same.

Reply:
Right, same response about how 12.3 is a point release of a supported OS